README.md

```diff
@@ -197,3 +207,16 @@ Object with the default options for the notifications
 
 ## defaultOptionsByType
 Object with the default options for the notifications for specific types
+
+## Building
+
+_OS X users, ensure you have `wget` installed (available in `brew`)_
+
+To run the tests:
+
+```bash
+$ ./workaround.sh
```

Comment:
Thanks, I ran that and then loaded it up in the browser. Getting errors.
Will work on fixing them (although strange that Travis doesn't throw
errors). I'll also update the README with that command you just gave me.


On 06/01/2015 18:06, Marco de Jongh wrote:
> In README.md
> <https://github.com/gfk-ba/meteor-notifications/pull/20#discussion-diff-22516300>:
> 
>> @@ -197,3 +207,16 @@ Object with the default options for the notifications
>>  
>>  ## defaultOptionsByType
>>  Object with the default options for the notifications for specific types
>> +
>> +## Building
>> +
>> +_OS X users, ensure you have `wget` installed (available in `brew`)_
>> +
>> +To run the tests:
>> +
>> +```bash
>> +$ ./workaround.sh
> 
> @hiddentao <https://github.com/hiddentao> Yes thats because you are
> running the script ment for travis.
> I'd recommend you to just use |meteor test-packages ./|
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gfk-ba/meteor-notifications/pull/20/files#r22516300>.
> 

-- 
Ramesh Nair

Freelance Web Developer

www.hiddentao.com
www.github.com/hiddentao